### PR TITLE
adding my first file

### PR DESCRIPTION
**Pull request by Rbncrnj**
yo, working on my first file with RAY please approve thanks

### CODE REVIEW

File: ruben_first_file.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# adrienne, this is rubens first file
```
Comment (raymendozajr) [inline ruben_first_file.py]: Yo Ruben, let's use some proper capitalization, possessive apostrophes, and period please!  I am a linguist! :p

**Review by raymendozajr — CHANGES_REQUESTED** [review]

File: ruben_first_file.py (hunk 1 above)
Comment (Rbncrnj) [reply ruben_first_file.py]: Check out my changes.

Comment (raymendozajr) [reply ruben_first_file.py]: Sounds good! LGTM!

**Review by raymendozajr — APPROVED** [review]
LGTM!

**Pull request merged**